akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala

```diff
@@ -116,21 +116,6 @@ final class ClusterSettings(val config: Config, val systemName: String) {
     cc.getMillisDuration(key).requiring(_ >= Duration.Zero, key + " >= 0s")
   }
 
-  // specific to the [[akka.cluster.DefaultDowningProvider]]
-  val AutoDownUnreachableAfter: Duration = {
-    val key = "auto-down-unreachable-after"
-    toRootLowerCase(cc.getString(key)) match {
-      case "off" => Duration.Undefined
-      case _     => cc.getMillisDuration(key).requiring(_ >= Duration.Zero, key + " >= 0s, or off")
-    }
-  }
-
-  /**
-   * @deprecated Specific to [[akka.cluster.AutoDown]] should not be used anywhere else, instead
-   *   ``Cluster.downingProvider.downRemovalMargin`` should be used as it allows the downing provider to decide removal
-   *   margins
-   */
-  @deprecated("Use Cluster.downingProvider.downRemovalMargin", since = "2.4.5")
```

Comment:
Isn't the deprecation still valid?

Reply:
A downing provider can define a `downRemovalMargin`, but if no downing provider is used this is still useful to have as a possible config property. It is used from the `NoDowning` provider and also from ClusterSingleton. I think we should just revert the decision to deprecate.